frontend/src/pages/RunDetailsV2.tsx

```diff
@@ -294,14 +294,23 @@ function updateToolBarActions(
 }
 
 function getDetailsFields(apiRun?: ApiRun): Array<KeyValue<string>> {
+  // check if the run has finished or not. The default value for apiRun.finished_at is
+  // Date(0), when it is not specified.
+  var finished_at = new Date(0);
+  if (apiRun?.finished_at) {
+    finished_at = new Date(apiRun?.finished_at);
```

Comment:
We can omit ? here: `finished_at = new Date(apiRun.finished_at);`

Reply:
You are right, I will change this.